spec/services/atmosphere/billing_service_spec.rb

```diff
@@ -162,6 +193,41 @@
 
   end
 
+  context 'os_families' do
+    let(:windows_flavor) { create(:virtual_machine_flavor, compute_site: cs)}
+    let(:linux_flavor) { create(:virtual_machine_flavor, compute_site: cs)}
+
+    let(:poor_fund) { create(:fund, balance: 75, overdraft_limit: 0, compute_sites: [cs] )}
+
+    let(:config_inst) { create(:appliance_configuration_instance) }
+
+    let!(:windows_appl_type) { create(:not_shareable_appliance_type, os_family: windows_osfamily) }
```

Comment:
`let`s are reasonable when reusing in many specs, here they are used only for one spec,  instead of defining `let` you can use local variables.